components/widgets/admin_console/admin_panel.tsx

```diff
@@ -23,7 +23,7 @@ type Props = {
 
 const AdminPanel: React.FC<Props> = (props: Props) => (
     <div
-        className={'AdminPanel ' + props.className}
+        className={'AdminPanel clearfix ' + props.className}
         id={props.id}
     >
         <div
```

Comment:
This change was required because the current implementation of `AdminPanel` was applying a version of clearfix that caused overflow content to be hidden... with the filter component we dont want that since otherwise the filter could overflow out of the admin panel and not be visible